src/presentation/http/router/note.ts

```diff
@@ -77,46 +80,35 @@ const NoteRouter: FastifyPluginCallback<NoteRouterOptions> = (fastify, opts, don
     /**
      * Check if note does not exist
      */
-    if (!note) {
-      const response: ErrorResponse = {
-        status: StatusCodes.NOT_FOUND,
-        message: 'Note not found',
-      };
-
-      return reply.send(response);
+    if (note === null) {
```

Comment:
May be we can use here `isEmpty()` function?

Reply:
`getNoteById` declares that it can return only `Note` or `null`, so I think it's better to check only for `null` here to explicitly cover that case. We don't need to check for `undefined` or `''` or `{}` because that would mean that `getNoteById` works wrong or has wrong return type declarations